CMakeLists.txt

```diff
@@ -12,6 +12,12 @@ foreach(p
   endif()
 endforeach()
 
+set(BENCHMARK_INSTALL_BINDIR "bin" CACHE STRING "Installation directory for executables")
+set(BENCHMARK_INSTALL_INCLUDEDIR "include" CACHE STRING "Installation directory for headers")
+set(BENCHMARK_INSTALL_LIBDIR "lib" CACHE STRING "Installation directory for libraries")
+set(BENCHMARK_INSTALL_CMAKEDIR "lib/cmake/${PACKAGE_NAME}" CACHE STRING "Installation directory for cmake config files")
+set(BENCHMARK_INSTALL_PKGCONFIG "lib/pkgconfig" CACHE STRING "Installation directory for pkgconfig config files")
```

Comment:
I think `BENCHMARK_INSTALL_PKGCONFIG` is `${CMAKE_INSTALL_LIBDIR}/lib/pkgconfig`,
same with `BENCHMARK_INSTALL_CMAKEDIR`